python/python/bystro/beanstalkd/worker.py

```diff
@@ -41,15 +42,46 @@
 logger = logging.getLogger(__name__)
 
 
+class FunctionWrapper:
```

Comment:
Python ProcessPoolExecutor cannot serialize (to send to other processes) functions that are not defined as top-level functions (e.g. defined in this module, or imported from another module); the functions we're executing are instead passed to the listen() function.

To get around that, we use a custom serializer: cloud pickle, which is the serializer used by Ray, for the same purpose.